tests/Platform/Bridge/Google/Embeddings/EmbeddingsModelClientTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Tests\Platform\Bridge\Google\Embeddings;
+
+use PhpLlm\LlmChain\Platform\Bridge\Google\Embeddings;
+use PhpLlm\LlmChain\Platform\Bridge\Google\Embeddings\ModelClient;
+use PhpLlm\LlmChain\Platform\Response\VectorResponse;
+use PhpLlm\LlmChain\Platform\Vector\Vector;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Small;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\Attributes\UsesClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+#[CoversClass(ModelClient::class)]
+#[Small]
+#[UsesClass(Vector::class)]
+#[UsesClass(VectorResponse::class)]
+#[UsesClass(Embeddings::class)]
+class EmbeddingsModelClientTest extends TestCase
+{
+    #[Test]
+    public function itRequests(): void
+    {
+        $response = $this->createStub(ResponseInterface::class);
+        $response
+            ->method('toArray')
+            ->willReturn(json_decode($this->getEmbeddingStub(), true));
+
+        $httpClient = self::createMock(HttpClientInterface::class);
```

Comment:
Also stub? We don't expect anything on it

Reply:
Now we actually expect something – it was quite dummy test before